etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/EtorMetadataStep.java

```diff
@@ -8,5 +8,6 @@ public enum EtorMetadataStep {
 
     CONTACT_SECTION_ADDED_TO_PATIENT,
 
-    SENT_TO_REPORT_STREAM
+    SENT_TO_REPORT_STREAM,
+    ETOR_PROCESSING_TAG_ADDED_TO_MESSAGE_HEADER
```

Comment:
Are these in order? If so can we perhaps add a comment to the java docs saying so and asking future devs to strive to maintain the order

Reply:
Good idea, I'll put them in order.